percona-xtradb-cluster-8.0-backup/recovery-cloud.sh

```diff
@@ -42,8 +42,8 @@ destination() {
 	fi
 }
 
-xbcloud get ${XBCLOUD_ARGS} "$(destination).sst_info" --parallel=10 | xbstream -x -C "${tmp}" --parallel="$(grep -c processor /proc/cpuinfo)"
-xbcloud get ${XBCLOUD_ARGS} "$(destination)" --parallel=10 | xbstream --decompress -x -C "${tmp}" --parallel="$(grep -c processor /proc/cpuinfo)"
+xbcloud get ${XBCLOUD_ARGS} "$(destination).sst_info" --parallel=10 | xbstream -x -C "${tmp}" --parallel="$(grep -c processor /proc/cpuinfo)" $XBSTREAM_EXTRA_ARGS
```

Comment:
I think we need to improve `--parallel=` option. I do not know why we set '10'. We can get it from  /proc/cpuinfo.
```suggestion
xbcloud get ${XBCLOUD_ARGS} "$(destination).sst_info" --parallel="$(grep -c processor /proc/cpuinfo)" | xbstream -x -C "${tmp}" --parallel="$(grep -c processor /proc/cpuinfo)" $XBSTREAM_EXTRA_ARGS
```

Reply:
https://github.com/percona/percona-docker/pull/869/commits/9f826ce94b7cdf2ebcc437c0c03d01356198efbd